src/Cache/Cache.php

```diff
@@ -362,6 +383,37 @@ public static function read($key, $config = 'default')
         return $engine->read($key);
     }
 
+    /**
+     * Fetches a value from the cache.
+     *
+     * ### Usage:
+     *
+     * Reading from the active cache configuration.
+     *
+     * ```
+     * Cache::get('my_data');
+     * ```
+     *
+     * Reading from a specific cache configuration.
+     *
+     * ```
+     * Cache::get('my_data', null, 'long_term');
+     * ```
+     *
+     * @param string $key The unique key of this item in the cache.
+     * @param mixed $default Default value to return if the key does not exist.
+     * @param string $config optional name of the configuration to use. Defaults to 'default'
+     * @return mixed The value of the item from the cache, or $default in case of cache miss.
+     * @throws \Psr\SimpleCache\InvalidArgumentException
+     *   MUST be thrown if the $key string is not a legal value.
+     */
+    public static function get($key, $default = null, $config = 'default')
```

Comment:
For the new cache interface, should we consider making the engines implement PSR16, and `Cache` becoming more of a registry/factory/container for cache engines? Right now we can't easily inject caching objects into other things. Would it be 'better' if we used this new interface implementation as an opportunity to make the interface something like

```php
// Get an engine
$cache = Cache::engine('default');
// Use the engine
$cache->set($key, $value, $ttl);
$value = $cache->get($key, $default);
```

I recognize this would be a big departure from what we have today. I think it would let us make engines more standalone and set us up for a DI container in the future.

Reply:
Shouldn't this be `getEngine()`? I thought to implement PSR6 first, then 16 in a follow up PR.

Reply:
`Cache::engine()` only functions as a getter, so it doesn't require a `get` prefix to me.

Reply:
I think we should use getX() it's more consistent and there is no need if we ever add a getter to refactor it.